recordablesurfaceview/src/main/java/com/uncorkedstudios/android/view/recordablesurfaceview/RecordableSurfaceView.java

```diff
@@ -595,6 +610,9 @@ public void surfaceChanged(SurfaceHolder surfaceHolder, int i, int width, int he
 
         }
 
+        public void queueEvent(Runnable r) {
```

Comment:
Does this method serve a purpose?

Reply:
Good call. Superfluous definition that I missed in cleanup.